### ISSUE TITLE
Why is the accuracy so bad? how sovle？？？

### ISSUE THREAD

**Issue by nanian111** [jfzhang95/pytorch-deeplab-xception#120]
Acc:0.7699842521985277, Acc_class:0.836586056915853, mIoU:0.47887653449025225, fwIoU: 0.6562474380712945

**Comment by Wuyang9** [comment]
I also meet a similar situation. I used a graphics card to train on voc2012. After 50 training sessions, the effect was not significantly improved. The training training round was 150, and the mIoU was only about 0.59. Have you solved this problem?

**Comment by Wuyang9** [comment]
@nanian111

**Comment by Wuyang9** [comment]
@nanian111 
I also meet a similar situation. I used a gpu to train on voc2012. After 50 training sessions, the effect was not significantly improved. The training training round was 150, and the mIoU was only about 0.59. Have you solved this problem?

**Comment by johsnows** [comment]
me too.I get 0.41 mIoU on voc2012 after 50 trainning sessions, envn though i tryed laod pretrain model

**Comment by jfzhang95** [comment]
I have updated the code, please try again.
